### DISCUSSION TITLE
Logarithmic FFT?

### DISCUSSION THREAD

**Discussion by jajito** [pschatzmann/arduino-audio-tools#850] [Q&A] [answered]
Hi Phil, congratulations again for your work, keeps astonishing me every time.
Is there a way of having the results of the FFT in an logarithmic array of frequencies, instead of the linear ones?
The most part of audio information is in the low and mid register, and not in the high frequencies.
That's why usually the spectrum analysers displays the frequency information in a logarithmic scale, instead of a linear one.
Any way to do that with the AudioRealFFT or any of the others FFT?
Thanks for your job!

**Comment by pschatzmann** [comment, marked as answer]
I am not aware of any FFT implementation that would do that. 
If you find one fee free to extend the framework and submit a pull request...

**Comment by jajito** [comment]
Thanks Phil.
I have found an approach to the subject, and will try.
If I'm successful I will let you know...
